changelog/unreleased/print-resolved-listen-addr

```diff
@@ -0,0 +1,15 @@
+Enhancement: after start-up, server served message now prints actual listen address
```

Comment:
That changelog entry belongs to https://github.com/restic/rest-server/pull/271 . The name of changelog files **must** either start with `issue-` or `pull-` followed by the issue/PR number. That is, it should be called `pull-271`.

Reply:
Yes, I'll add this changelog entry to that PR and name as requested.